Formula/easyengine.rb

```diff
@@ -24,6 +24,26 @@ def install
     bin.install "easyengine.phar" => "ee"
   end
 
+  def caveats
```

Comment:
This is not specific to Homebrew so this needs to be in the upstream documentation.